packages/datx-jsonapi-angular/projects/datx-jsonapi-angular/test/general.test.ts

```diff
@@ -629,16 +629,19 @@ describe('General', () => {
     expect(user).not.toBeNull();
 
     if (user) {
-      expect(user['selected']).toBeUndefined();
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      expect((user as any)['selected']).toBeUndefined();
```

Comment:
this doesn't seem ok... looks like all model types are incorrect?

Reply:
Similar as the previous comment, this is due to stricter typechecking after migrating to angular-cli and it's out-of-the-box tsconfig. I will disable some of those checks so that it's more in-line with what we had before

Reply:
is this promise/observable related? If not, I might want to look into this and try to fix types.

Reply:
I don't think so, I think it was only because of some different tsconfig values. I have reverted them to be more lenient like they were before and removed casting to any (the comment is now outdated)